src/lib/seam/components/AccessCodeTable/AccessCodeHealthBar.tsx

```diff
@@ -0,0 +1,68 @@
+import { CheckIcon } from 'lib/icons/Check.js'
+import { ExclamationCircleOutlineIcon } from 'lib/icons/ExclamationCircleOutline.js'
+import type { UseAccessCodesData } from 'lib/seam/access-codes/use-access-codes.js'
+import { TableFilterBar } from 'lib/ui/Table/TableFilterBar/TableFilterBar.js'
+import { TableFilterItem } from 'lib/ui/Table/TableFilterBar/TableFilterItem.js'
+
+export type AccessCodeFilter = 'access_code_issues'
+
+interface AccessCodeHealthBarProps {
+  accessCodes: Array<UseAccessCodesData[number]>
+  filter: AccessCodeFilter | null
+  onFilterSelect: (filter: AccessCodeFilter | null) => void
+}
+
+export function AccessCodeHealthBar({
+  accessCodes,
+  filter,
+  onFilterSelect,
+}: AccessCodeHealthBarProps): JSX.Element {
+  const erroredAccessCodes = accessCodes.filter(
+    (accessCodes) => (accessCodes?.errors?.length ?? 0) > 0
```

Comment:
Should this be `(accessCode) => (accessCode?.errors?.length ?? 0) > 0`

We may also want to remove the undefined operator unless we want to push this through before the SDK/type updates

CC @razor-x

Reply:
New sdk should be on main now with this, so can be removed if the branch is updated.